Unable to install package on Ubuntu/Linux; old sqlite3 version dependency?

**Park**
Which park/resort has this issue?: `N/A`

**Context**
- themeparks version used: `5.1.52`
- Operating System:
```
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal
```

**Describe the bug**
Running `npm install themeparks --save` on Linux (specifically Ubuntu) fails. It seems like the failure is caused by currently using an older version of `sqlite3`. Ran an experiment and bumped the dependency to 5.0.2+ and installation completed successfully.

**Output**
```
npm WARN deprecated socks@1.1.10: If using 2.x branch, please upgrade to at least 2.1.6 to avoid a serious bug with socket data flow and an import issue introduced in 2.1.0
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated node-pre-gyp@0.11.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future
npm ERR! code 1
npm ERR! path /tmp/node_modules/sqlite3
npm ERR! command failed
npm ERR! command sh -c node-pre-gyp install --fallback-to-build
npm ERR! make: Entering directory '/tmp/node_modules/sqlite3/build'
npm ERR!   ACTION deps_sqlite3_gyp_action_before_build_target_unpack_sqlite_dep Release/obj/gen/sqlite-autoconf-3310100/sqlite3.c
npm ERR! make: Leaving directory '/tmp/node_modules/sqlite3/build'
npm ERR! Failed to execute '/usr/bin/node /usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/tmp/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node --module_name=node_sqlite3 --module_path=/tmp/node_modules/sqlite3/lib/binding/node-v93-linux-x64 --napi_version=8 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v93' (1)
npm ERR! node-pre-gyp info it worked if it ends with ok
npm ERR! node-pre-gyp info using node-pre-gyp@0.11.0
npm ERR! node-pre-gyp info using node@16.13.0 | linux | x64
npm ERR! node-pre-gyp WARN Using needle for node-pre-gyp https download 
npm ERR! node-pre-gyp info check checked for "/tmp/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node" (not found)
npm ERR! node-pre-gyp http GET https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v4.2.0/node-v93-linux-x64.tar.gz
npm ERR! node-pre-gyp http 403 https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v4.2.0/node-v93-linux-x64.tar.gz
npm ERR! node-pre-gyp WARN Tried to download(403): https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v4.2.0/node-v93-linux-x64.tar.gz 
npm ERR! node-pre-gyp WARN Pre-built binaries not found for sqlite3@4.2.0 and node@16.13.0 (node-v93 ABI, glibc) (falling back to source compile with node-gyp) 
npm ERR! node-pre-gyp http 403 status code downloading tarball https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v4.2.0/node-v93-linux-x64.tar.gz 
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@8.2.0
npm ERR! gyp info using node@16.13.0 | linux | x64
npm ERR! gyp info ok 
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@8.2.0
npm ERR! gyp info using node@16.13.0 | linux | x64
npm ERR! gyp info find Python using Python version 3.8.5 found at "/usr/bin/python3"
npm ERR! gyp info spawn /usr/bin/python3
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/usr/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/tmp/node_modules/sqlite3/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/usr/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/root/.cache/node-gyp/16.13.0/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/root/.cache/node-gyp/16.13.0',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/usr/lib/node_modules/npm/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/root/.cache/node-gyp/16.13.0/<(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/tmp/node_modules/sqlite3',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp info ok 
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@8.2.0
npm ERR! gyp info using node@16.13.0 | linux | x64
npm ERR! gyp info spawn make
npm ERR! gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
npm ERR! /bin/sh: 1: python: not found
npm ERR! make: *** [deps/action_before_build.target.mk:13: Release/obj/gen/sqlite-autoconf-3310100/sqlite3.c] Error 127
npm ERR! gyp ERR! build error 
npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
npm ERR! gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:390:28)
npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)
npm ERR! gyp ERR! System Linux 5.4.0-28-generic
npm ERR! gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/tmp/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node" "--module_name=node_sqlite3" "--module_path=/tmp/node_modules/sqlite3/lib/binding/node-v93-linux-x64" "--napi_version=8" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v93"
npm ERR! gyp ERR! cwd /tmp/node_modules/sqlite3
npm ERR! gyp ERR! node -v v16.13.0
npm ERR! gyp ERR! node-gyp -v v8.2.0
npm ERR! gyp ERR! not ok 
npm ERR! node-pre-gyp ERR! build error 
npm ERR! node-pre-gyp ERR! stack Error: Failed to execute '/usr/bin/node /usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/tmp/node_modules/sqlite3/lib/binding/node-v93-linux-x64/node_sqlite3.node --module_name=node_sqlite3 --module_path=/tmp/node_modules/sqlite3/lib/binding/node-v93-linux-x64 --napi_version=8 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v93' (1)
npm ERR! node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/tmp/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
npm ERR! node-pre-gyp ERR! stack     at ChildProcess.emit (node:events:390:28)
npm ERR! node-pre-gyp ERR! stack     at maybeClose (node:internal/child_process:1064:16)
npm ERR! node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)
npm ERR! node-pre-gyp ERR! System Linux 5.4.0-28-generic
npm ERR! node-pre-gyp ERR! command "/usr/bin/node" "/tmp/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
npm ERR! node-pre-gyp ERR! cwd /tmp/node_modules/sqlite3
npm ERR! node-pre-gyp ERR! node -v v16.13.0
npm ERR! node-pre-gyp ERR! node-pre-gyp -v v0.11.0
npm ERR! node-pre-gyp ERR! not ok

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-11-26T19_21_57_427Z-debug.log
```

[2021-11-26T19_21_57_427Z-debug.log](https://github.com/cubehouse/themeparks/files/7610447/2021-11-26T19_21_57_427Z-debug.log)

--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
     "needle": "^2.6.0",
     "random-useragent": "^0.3.1",
     "socks-proxy-agent": "^3.0.1",
-    "sqlite3": ^5.0.2",
+    "sqlite3": "^5.0.2",
     "uuid": "^3.4.0",
     "xml2js": "^0.4.23",
     "yauzl": "^2.10.0"

--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
     "needle": "^2.6.0",
     "random-useragent": "^0.3.1",
     "socks-proxy-agent": "^3.0.1",
-    "sqlite3": "^5.0.2"
+    "sqlite3": "^5.0.2",
     "uuid": "^3.4.0",
     "xml2js": "^0.4.23",
     "yauzl": "^2.10.0"

--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
     "needle": "^2.6.0",
     "random-useragent": "^0.3.1",
     "socks-proxy-agent": "^3.0.1",
-    "sqlite3": "^4.2.0",
+    "sqlite3": "^5.0.2
     "uuid": "^3.4.0",
     "xml2js": "^0.4.23",
     "yauzl": "^2.10.0"

--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
     "needle": "^2.6.0",
     "random-useragent": "^0.3.1",
     "socks-proxy-agent": "^3.0.1",
-    "sqlite3": ^5.0.2r",
+    "sqlite3": ^5.0.2",
     "uuid": "^3.4.0",
     "xml2js": "^0.4.23",
     "yauzl": "^2.10.0"

--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
     "needle": "^2.6.0",
     "random-useragent": "^0.3.1",
     "socks-proxy-agent": "^3.0.1",
-    "sqlite3": "github:https://github.com/mapbox/node-sqlite3#master",
+    "sqlite3": ^5.0.2r",
     "uuid": "^3.4.0",
     "xml2js": "^0.4.23",
     "yauzl": "^2.10.0"

--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
     "needle": "^2.6.0",
     "random-useragent": "^0.3.1",
     "socks-proxy-agent": "^3.0.1",
-    "sqlite3": "^5.0.2
+    "sqlite3": "^5.0.2"
     "uuid": "^3.4.0",
     "xml2js": "^0.4.23",
     "yauzl": "^2.10.0"

--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
     "needle": "^2.6.0",
     "random-useragent": "^0.3.1",
     "socks-proxy-agent": "^3.0.1",
-    "sqlite3": "^5.0.2",
+    "sqlite3": "github:https://github.com/mapbox/node-sqlite3#master",
     "uuid": "^3.4.0",
     "xml2js": "^0.4.23",
     "yauzl": "^2.10.0"

